src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/EqualityConstraintValidation107.feature.disabled

```diff
@@ -0,0 +1,126 @@
+Feature: Equality Constraint Validation
+
+        Background:
+            Given this school exists
+                  | schoolId  | nameOfInstitution      | gradeLevels     | educationOrganizationCategories |
+                  | 255901001 | Grand Bend High School | ["Ninth grade"] | ["School"]                      |
+
+        @ignore
+        Scenario: Making a Post request when value does not match the same value in an array
+             When a POST request is sent to "/ed-fi/sections" with body
```

Comment:
This should be `When a POST request is made to "<url>" with`

Reply:
Complete